src/main/java/org/maxgamer/quickshop/integration/plotsquared/PlotSquaredIntegrationV5.java

```diff
@@ -90,12 +101,47 @@ public void load() {
         GlobalFlagContainer.getInstance().addAll(Arrays.asList(createFlag, tradeFlag));
         plugin.getLogger().info(ChatColor.GREEN + getName() + " flags register successfully.");
         Util.debugLog("Success register " + getName() + " flags.");
+        new PlotAPI().registerListener(this);
     }
```

Comment:
Since we are using plot api, we need to unregister from it when unloading

Reply:
BTW there no unregisterListener or something like that